tests/mvp-demo/mvp-demo.go

```diff
@@ -260,7 +261,7 @@ var _ = framework.MvpDemoSuiteDescribe("MVP Demo tests", Label("mvp-demo"), func
 					return false
 				}
 				return pipelineRun.HasStarted()
-			}, pipelineRunStartedTimeout, defaultPollingInterval).Should(BeTrue())
+			}, pipelineRunStartedTimeout, defaultPollingInterval).Should(BeTrue(), fmt.Sprintf("component %q in namespace %q did not start a pipelinerun named %q with an annotation naming the namespace %q in time", componentName, managedNamespace, release.Name, release.Namespace))
```

Comment:
Shouldn't be `pipelineRun.Name`?

```suggestion
                        }, pipelineRunStartedTimeout, defaultPollingInterval).Should(BeTrue(), fmt.Sprintf("component %q in namespace %q did not start a pipelinerun named %q with an annotation naming the namespace %q in time", componentName, managedNamespace, pipelineRun.Name, release.Namespace))
```

Reply:
yeah it does a list with a label to the release name

that said, I should probably include both names in case they don't have a common prefix

I will update that way vs. your suggested commit, then fix the typos you identified manually

thanks